SolarCalcs/Calculate_n.py

```diff
@@ -0,0 +1,13 @@
+#Calculate n. The numbered day of the year, for example January 1st is n=1.
+import datetime
+import pytest
+
+def nth_day(year, month, day):
+	mydate=datetime.date(year, month, day)
+	return mydate.strftime('%j')
+
```

Comment:
You might want to throw a ValueError if the user inputs wrong values. This is separate from the tests.
Some things to catch:
non-integers, negative values, out-of-bounds values, proper date (e.g. MM-DD-YYYY vs DD-MM-YYYY)